src/main/resources/templates/aeolus/c_plus_plus/default_static.sh

```
@@ -0,0 +1,78 @@
+
+static_code_analysis () {
+  echo '⚙️ executing static_code_analysis'
+  #!/usr/bin/env bash
+
+  # ------------------------------
+  # Task Description:
+  # Check the student code for common issues
+  # ------------------------------
+
+  ln -s build/compile_commands.json .
+  clang-tidy "${studentParentWorkingDirectoryName}"/**/*.{c,h,cpp,hpp} | clang-tidy-sarif --output test-reports/clang-tidy.sarif
```

Shellcheck (0.10.0)

[warning] studentParentWorkingDirectoryName is referenced but not assigned.

(SC2154)